app/uk/gov/hmrc/icedsubscriptionfrontend/views/BadUserIndividualPage.scala.html

```diff
@@ -0,0 +1,48 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.icedsubscriptionfrontend.controllers
+@import uk.gov.hmrc.icedsubscriptionfrontend.views.html.helpers._
+
+@this(layout: Layout,
+        p: p,
+        paragraph: paragraph,
+        heading2: heading2,
+        link: link)
+
+@()(implicit messages: Messages, requestHeader: RequestHeader)
+@layout(pageTitle = Some(messages("service.pageTitle.format", messages("individual.heading"))), showSignOutLink = true, showTimeoutDialog = true) {
+
+    @heading2("individual.heading")
```

Comment:
Test fails: inject an  `HmrcPageHeading` and use
```suggestion    
    @hmrcPageHeading(PageHeading(text = messages("individual.heading")))
```